recipes/ffbidx/build.sh

```diff
@@ -0,0 +1,9 @@
+PKG_CONFIG_PATH=${BUILD_PREFIX}/share/pkgconfig:${PKG_CONFIG_PATH}
```

Comment:
If you move the dependencies into host, you should no longer need this
```suggestion
```

Reply:
Done